rl4co/models/nn/mlp.py

```diff
@@ -1,14 +1,19 @@
-from typing import List
+from typing import List, Union
 
 import torch.nn as nn
 
+from rl4co.utils.pylogger import get_pylogger
+
+log = get_pylogger(__name__)
+
 
 class MLP(nn.Module):
     def __init__(
         self,
         input_dim: int,
         output_dim: int,
         num_neurons: List[int] = [64, 32],
+        dropout_probs: Union[None, List[float]] = None,
```

Comment:
[Minor] Note that we could also use the MLP from TorchRL, but no need to change now since here we can add more custom stuff

Reply:
Hi @fedebotu, thanks for the comments! I have added the tests for N2S and resolved the conflicts!